.github/workflows/coverage.yml

```
@@ -3,36 +3,103 @@ name: Coverage Report - dev
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+      - name: Install nextest
+        uses: taiki-e/install-action@nextest
+      - name: Install llvm-cov
+        run: cargo install cargo-llvm-cov && sudo apt install -y lcov
+      - name: Install dependencies
+        run: sudo apt-get install clang
+      - uses: actions/setup-python@v5
+        with:
+          python-version: '3.11'
+      - name: Install dependencies
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:1:67: Double quote to prevent globbing and word splitting

(shellcheck)